docs/webui_tutorial.rst

```diff
@@ -0,0 +1,74 @@
+Web Application Tutorial
+############################
+
+.. toctree::
+  :maxdepth: 2
+
+In order to quickly give people an overview and idea of what the Raiden Developer Preview is capable of, a simple web application has been created. This application utilizes the :doc:`Raiden REST API endpoints <rest_api>` to allow the user to interact with tokens networks, transfer tokens, see the current status of open channels along with closing and settling channels to name some of the functionalities. For a more specific guide of the API itself see the :doc:`API Walkthrough <api_walkthrough>`.
+
+The main focus of the web application is to display functionality and not to look pretty. Should you however want to make it pretty, don't hesitate to create a `pull request <https://github.com/raiden-network/raiden/pulls>`_.
+
+
+Running the web application
+---------------------------
+There are a few simple steps that needs to be carried out in order to run the Raiden web application. Firstly, a Raiden node needs to be setup. See :doc:`overview_and_guide` for instructions on this.
+
+The only dependency needed to run the web application is `node.js <https://docs.npmjs.com/getting-started/installing-node>`_.
+
+Once a Raiden node is up and running and node.js has been installed, it is quite easy to start the web application::
+
+    cd raiden/ui/web
+
+Followed by::
+
+    npm install
+
+And then to run the application simply do::
```

Comment:
Actually, this documentation section needs to be changed: you don't need to install and run npm dependencies by it's own, and it's actually not recommended, as you'd run another webserver, and requires to enable CORS in Raiden.
The recommended steps are:
- Before installing raiden, go to the Raiden root folder and run: `python setup.py compile_webui` (this is not run automatically to avoid increasing test times)
- Run Raiden and, in your browser, go to `http://localhost:5001`, or whatever you set in `--api-address`, and it should serve the compiled (production) files directly, instead of a development webserver. To disable raiden from serving this webapp, you can pass `--no-web-ui` to it.